### ISSUE TITLE
dep ensure -add .com/aws/aws-xray-sdk-go does not working

### ISSUE THREAD

**Issue by k-kurikuri** [aws/aws-xray-sdk-go#112]
Hi. The dep command generated an error when following the README.

```
$ dep ensure -add github.com/aws/aws-xray-sdk-go
Fetching sources...

Solving failure: No versions of github.com/aws/aws-xray-sdk-go met constraints:
    v0.9.4: Could not introduce github.com/aws/aws-xray-sdk-go@v0.9.4, as its subpackage github.com/aws/aws-xray-sdk-go does not contain usable Go code (*build.NoGoError)..
```

This seems to happen because there is no `.go` under the root package.
The package containing `.go` was successful.

```
$ dep ensure-add github.com/aws/aws-xray-sdk-go/xray
```

aws-xray-sdk will probably work with this command.

**Comment by luluzhao** [comment]
@k-kurikuri , thank you for contacting us regarding your issue. It seems like this is an [issue](https://github.com/golang/dep/issues/1306) with `dep` and currently there is no fix for it. Here are two solutions to fix this. First one like you mentioned above, install the individual packages using `dep ensure -add`. Second one is inside your Gopkg.toml file and then write down the dependency you want to install like below
```
[[constraint]]
   name = "github.com/aws/aws-xray-sdk-go"
   version = "v1.0.0-rc.11"
```
After finishing this, run `dep ensure` to install the dependency. 

Let me know if you have any questions.

**Comment by k-kurikuri** [comment]
@luluzhao , Thank you for your reply. It solved by adding `v1.0.0-rc.11` to `Gopkg.toml`. I will implement using v1.0.0-rc version.
